File: CS/TestApplication/MyWebDataStore.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Web.Services;

namespace TestApplication {
    // TODO: change MyWebDataSource namespace to match the published service's namespace
    [WebServiceBinding(Namespace = WebServiceAttribute.DefaultNamespace)]
    class MyWebDataStore : DevExpress.Xpo.WebServiceDataStore {
        public MyWebDataStore(string url)
            : base(url, DevExpress.Xpo.DB.AutoCreateOption.SchemaAlreadyExists) {
        }
    }
}

File: VB/TestApplication/MyWebDataStore.vb
Imports System.Web.Services

Namespace TestApplication

    ' TODO: change MyWebDataSource namespace to match the published service's namespace
    <WebServiceBinding([Namespace]:=WebServiceAttribute.DefaultNamespace)>
    Friend Class MyWebDataStore
        Inherits DevExpress.Xpo.WebServiceDataStore

        Public Sub New(ByVal url As String)
            MyBase.New(url, DevExpress.Xpo.DB.AutoCreateOption.SchemaAlreadyExists)
        End Sub
    End Class
End Namespace
